### PR TITLE
support multi pb

### PR DESCRIPTION
**Pull request by gaoxinge**
**What this PR does**:

- support multi pb

**Which issue(s) this PR fixes**:

Fixes https://github.com/apache/dubbo-go/pull/1261#issuecomment-889575887

**Special notes for your reviewer**:

NONE

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: protocol/grpc/protoc-gen-dubbo/plugin/dubbo/dubbo.go [hunk 1]
```diff
@@ -311,6 +304,15 @@ func (g *dubboGrpc) generateServerMethod(servName, fullServName string, method *
 	}
 	streamType := unexport(servName) + methName + "Server"
 	g.P("func ", hname, "(srv interface{}, stream ", grpcPkg, ".ServerStream) error {")
```
Comment (LaurenceLiZhixin) [inline protocol/grpc/protoc-gen-dubbo/plugin/dubbo/dubbo.go:306]: 这里看着像插到函数里了，这里编译后不会报错么？

**Comment by LaurenceLiZhixin** [comment]
Thanks for your improvment!
Why you add  proto files to grpc/internal?  I can't find usage for them.
And pls add an example to samples if needed. @gaoxinge

**Comment by gaoxinge** [comment]
Proto files is used to test that there is no confliction when generating multi pb.go in one package, not for grpc test. 

Refer to https://github.com/apache/dubbo-go/pull/1261#issuecomment-889575887 .

**Comment by LaurenceLiZhixin** [comment]
Did you test locally using stub generated by new protoc-gen-dubbo tools? @gaoxinge

**Comment by gaoxinge** [comment]
@LaurenceLiZhixin Yes, I have test this locally.

**Review by LaurenceLiZhixin — APPROVED** [review]

**Review by AlexStocks — APPROVED** [review]

**Pull request merged**